xc7/archs/zynq7/tiles/bram_l/CMakeLists.txt

```diff
@@ -3,4 +3,5 @@ project_xray_tile(
   TILE BRAM_L
   SITE_TYPES BRAM_L/BRAM_L
   FUSED_SITES
+  EQUIVALENT_SITES BRAM_L
```

Comment:
Let's file an issue about fixing the `BRAM_[LR]` equivalence to track it.

Reply:
You mean that eventually we will have only a single `BRAM` tile which will hold the `BRAM_L` and `BRAM_R`?

Reply:
Ya